datahub/interaction/email_processors/processors.py

```
@@ -59,6 +67,17 @@ class CalendarInteractionEmailProcessor(EmailProcessor):
     instance for it if the information is valid.
     """
 
+    def _notify_meeting_ingest_failure(self, message, errors):
+        try:
+            sender_email = message.from_[0][1]
+        except IndexError:
+            return
```

Do we need to be notified if this happens?

---

I think the user cannot be notified in cases where we cannot find the sender's (dit-adviser) email. But I agree, we should at least log this.

---

Let's log these. We shouldn't typically be silently failing.